### PR TITLE
Fix surface swi simulations

### PR DESCRIPTION
**Pull request by dasvyat**
New evaluator for surface water content

### CODE REVIEW

**Comment by ecoon** [comment]
@dasvyat I'm not sure I understand this PR.  Why are there all these extra commits that I made months and months ago?  Many of them look like they're already in ats-1.2; are they not in master?

**Review by ecoon — CHANGES_REQUESTED** [review]
Ok, so it looks like the only "real" change here is the new evaluator?  Can you just make only that change on konstantin/state and do a new PR?

File: .github/workflows/ats-ci.yml [hunk 1]
```diff
@@ -17,7 +17,8 @@ jobs:
       with:
         repository: amanzi/amanzi
         submodules: true
-    - name: Extract the branch name
+        ref:        amanzi-1.2
```
[the remark this answers is not in the document]
Comment (dasvyat) [reply .github/workflows/ats-ci.yml:20]: I didn't make this change it probably appeared due to merging

File: src/pks/flow/constitutive_relations/water_content/overland_pressure_unfrozen_water_content_evaluator.cc [hunk 2]
```diff
@@ -0,0 +1,136 @@
+/* -*-  mode: c++; indent-tabs-mode: nil -*- */
+
+/*
+  Evaluator for determining height( rho, head )
```
[the remark this answers is not in the document]
Comment (dasvyat) [reply src/pks/flow/constitutive_relations/water_content/overland_pressure_unfrozen_water_content_evaluator.cc:4]: Ethan, we discussed this issue a bit before. That was the name that we decided to use. In the previous version "surface_water_content_evaluator" you didn't use density and scaled by molar mass. It worked correctly for ice water case, but it is wrong for the salt case. We conclude that this one will not work for frozen case, so I called unfrozen_water_content

**Comment by ecoon** [comment]
closing this in favor of new, clean PR

**Pull request closed without merging**